ambassador/ambassador/ir/irauth.py

```diff
@@ -165,11 +143,21 @@ def config_dict(self):
             "cluster": self.cluster.name
         }
 
-        for key in [ 'allowed_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
+        for key in [ 'allowed_resquest_headers', 'allowed_authorization_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
             if self.get(key, None):
                 config[key] = self[key]
+        
+        # Sets request headers whitelist.
+        if self.get('allowed_resquest_headers', []):
```

Comment:
...and here and the line below. Whoops. :smile:

Reply:
Thanks for spotting the typos. I will fix this one too.